src/main/resources/locales/cy/case-name-search.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "Tanysgrifo gan ddefnyddio enw'r parti neu'r partïon sydd ynglŷn â'r achos ",
-  "heading": "Beth yw enw(au)’r parti neu bartïon sy’n rhan o’r achos?",
+  "title": "Tanysgrifo gan ddefnyddio enw'r achos",
```

Comment:
Translations just need reverting to previous ones for now as discussed, and will bundle the requests together in the new year

Reply:
Done